src/main/java/org/spongepowered/api/entity/living/player/Player.java

```diff
@@ -119,6 +120,14 @@
     boolean isChatColorsEnabled();
 
     /**
+     * Simulates a chat message from a player.
+     *
+     * @param message The message to send
+     * @return The event thrown from the message
+     */
+    MessageChannelEvent.Chat chat(Text message);
```

Comment:
This should accept `String` in my opinion. Clients send a `String` only, and this method should be used to simulate a client sending a `String` to the server.